test/mock/test_fake_backends.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Test of generated fake backends."""
+import math
+
+from qiskit import (QuantumRegister, ClassicalRegister, QuantumCircuit,
+                    schedule, transpile, assemble)
+from qiskit.pulse import Schedule
+from qiskit.qobj import PulseQobj
+from qiskit.test import QiskitTestCase
+from qiskit.test.mock.utils import ConfigurableBackend
+
+
+def get_test_circuit():
+    """Generates simple circuit for tests."""
+    desired_vector = [1 / math.sqrt(2), 0, 0, 1 / math.sqrt(2)]
+    qr = QuantumRegister(2, "qr")
+    cr = ClassicalRegister(2, 'cr')
+    qc = QuantumCircuit(qr, cr)
+    qc.initialize(desired_vector, [qr[0], qr[1]])
+    qc.measure(qr[0], cr[0])
+    qc.measure(qr[1], cr[1])
+    return qc
+
+
+class GeneratedFakeBackendsTest(QiskitTestCase):
+    """Generated fake backends test."""
+
+    def setUp(self) -> None:
+        self.backend = ConfigurableBackend("Tashkent", n_qubits=4)
+
+    def test_transpile_schedule_and_assemble(self):
```

Comment:
There is partial overlap here with: https://github.com/Qiskit/qiskit-terra/blob/master/test/python/providers/test_fake_backends.py#L51 but this is different enough that it's probably fine. It might be a good follow up to expand this or the other to cover both use cases and deduplicate things a bit.